src/index.ejs

```diff
@@ -204,8 +204,8 @@
 				<div id="start"><button type="submit"></button><span>Start</span></div>
 			</form>
 
-			<div id="fullscreen">
-				<div></div><span class="fullscreen__title">Fullscreen</span>
```

Comment:
This text was replaced with an HTML title attribute. It was not looking good with in game backgrounds.

Reply:
This was actually only used in the pre-match screen (similar to the other icons), while during combat it had no text at all.

Reply:
I see. Should I keep it?